site/layouts/shortcodes/feature-state.html

```diff
@@ -0,0 +1,13 @@
+{{ $valid_states := "alpha, beta, deprecated, stable" }}
+{{ $project := .Get "project" | default "Kueue" }}
+{{ $state := .Get "state" }}
+{{ $for_version := .Get "for_version" | default (.Page.Param "version")}}
+{{ $is_valid := strings.Contains $valid_states $state }}
+
+{{ if not $is_valid }}
+{{ errorf "%q is not a valid feature-state, use one of %q" $state $valid_states }}
+{{ else }}
+<div class="feature-state-notice feature-{{ $state }}">
+  <span class="feature-state-name">FEATURE STATE</span> <code>{{ $project }} {{ $for_version }} [{{ $state }}]</code>
```

Comment:
Try this:
```suggestion
  <span class="feature-state-name">Feature state</span>: {{ $state }} since {{ $project }} {{ $for_version }}
```

It's not exactly what's on the K8s main website, but I think it's close enough. We can use `text-transform` in the CSS if we want “FEATURE STATE” in all caps (I don't think it helps, TBH).